content/docs/alerting/configuration.md

```diff
@@ -0,0 +1,348 @@
+[ client_url:  <tmpl_string> | default = '{{ template "pagerduty.default.clientURL" . }}' ]
+
+# A description of the incident.
+[ description: <tmpl_string> | default = '{{ template "pagerduty.default.description" .}}' ]
+
+# A set of arbitrary key/value pairs that provide further detail
+# about the incident.
+[ details: { <string>: <tmpl_string>, ... } | default = {
+  firing:       '{{ template "pagerduty.default.instances" .Alerts.Firing }}'
+  resolved:     '{{ template "pagerduty.default.instances" .Alerts.Resolved }}'
+  num_firing:   '{{ .Alerts.Firing | len }}'
+  num_resolved: '{{ .Alerts.Resolved | len }}'
+} ]
+```
+
+
+## Slack receiver `<slack_config>`
+
+Slack notifications are sent via [Slack webhooks](https://api.slack.com/incoming-webhooks).
+
+```
+# Whether or not to notify about resolved alerts.
+[ send_resolved: <boolean> | default = true ]
+
+# The Slack webhook URL.
+[ api_url: <string> | default = global.slack_api_url ]
+
+# The channel or user to send notifications to.
+channel: <tmpl_string>
+
+# API request data as defined by the Slack webhook API.
+[ color: <tmpl_string> | default = '{{ if eq .Status "firing" }}danger{{ else }}good{{ end }}' ]
+[ username: <tmpl_string> | default = '{{ template "slack.default.username" . }}'
+[ title: <tmpl_string> | default = '{{ template "slack.default.title" . }}' ]
+[ title_link: <tmpl_string> | default = '{{ template "slack.default.titlelink" . }}' ]
+[ pretext: <tmpl_string> | default = '{{ template "slack.default.pretext" . }}' ]
+[ text: <tmpl_string> | default = '{{ template "slack.default.text" . }}' ]
+[ fallback: <tmpl_string> | default = '{{ template "slack.default.fallback" . }}' ]
+```
+
+
+## OpsGenie receiver `<opsgenie_config>`
+
+OpsGenie notifications are sent via the [OpsGenie API](https://www.opsgenie.com/docs/web-api/alert-api).
+
+```
+# Whether or not to notify about resolved alerts.
+[ send_resolved: <boolean> | default = true ]
+
+# The API key to use when talking to the OpsGenie API.
+api_key: <string>
+
+# The host to send OpsGenie API requests to.
+[ api_host: <string> | default = global.opsgenie_api_host ]
+
+# A description of the incident.
+[ description: <tmpl_string> | default = '{{ template "opsgenie.default.description" . }}' ]
+# A backlink to the sender of the notification.
+[ source: <tmpl_string> | default = '{{ template "opsgenie.default.source" . }}' ]
+
+# A set of arbitrary key/value pairs that provide further detail
+# about the incident.
+[ details: { <string>: <tmpl_string>, ... } ]
+```
+
+
+## Webhook receiver `<webhook config>`
+
+The webhook receiver allows configuring a generic receiver. The Alertmanager
+will send HTTP POST requests in the following JSON format to the configured
+endpoint:
+
+```
+{
+  "version": "2",
+  "status": "<resolved|firing>",
+  "alerts": [
+    {
+      "labels": <object>,
+      "annotations": <object>,
+      "startsAt": "<rfc3339>",
+      "endsAt": "<rfc3339>"
+    },
+    ...
+  ]
+}
+```
+
```

Comment:
Two consecutive code blocks could be confusing, I'd put them the other way around so that the last above sentence can go between them.